CHANGELOG.md

```diff
@@ -4,9 +4,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
 
+- Feature: Remove `ownerLocation` from `ProjectCardComponent`.
```

Comment:
Du coup, je proposerai plutôt `Display ownerLocation only if the prop is provided` ici à la place de `Remove …`